lab_1_classify_by_unigrams/main.py

```diff
@@ -11,14 +11,36 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-
+    if isinstance(text, str) == True:
```

Comment:
To be sure that all arguments have types that are needed for a proper work of a function. Or will it work properly just with  if isinstance(text, str): ? It will return True, right? Will this be enough for a proper work of a function?

Reply:
I did early return and changed statements with ==True to ==False

Reply:
Have you actually changed such statements?

Reply:
Now ==True and ==False are deleted, fixed